runtimes/mlflow/mlserver_mlflow/runtime.py

```diff
@@ -122,10 +128,17 @@ def _sync_metadata(self) -> None:
             return
 
         self.inputs = to_metadata_tensors(
-            self._signature.inputs, prefix=DefaultInputPrefix
+            schema=self._signature.inputs, prefix=DefaultInputPrefix
         )
         self.outputs = to_metadata_tensors(
-            self._signature.outputs, prefix=DefaultOutputPrefix
+            schema=self._signature.outputs, prefix=DefaultOutputPrefix
+        )
+
+        if not self._settings.parameters:
+            self._settings.parameters = ModelParameters()
+
+        self._settings.parameters.content_type = to_model_content_type(
```

Comment:
are we possibly overwriting the setting that is provided by the user? should we have a warning?

Reply:
Good point, we probably should. Same with `inputs` and `outputs`, which are already overriden by the MLflow model signature. 

I'll add that to the PR.